config/config.go

```diff
@@ -1940,6 +1940,9 @@ func InitServer(ctx context.Context, currentServers server_structs.ServerType) e
 
 	// Sets up the server log filter mechanism
 	initFilterLogging()
+	
+	// Register callback for runtime log level changes
```

Comment:
If this is only exposed for the case of the tests, note that here.

If you don't need to export this for the logging tests, do not export this from the module unnecessarily.